packages/web/src/js/utils/index.ts

```diff
@@ -13,4 +13,5 @@ export { reflow };
 export * from './ComponentFunctions';
 export * from './Deprecations';
 export * from './Elements';
+export { default as ScrollControl } from './ScrollControl';
```

Comment:
Why not `*`? I mean, alhough this is my personal preference, maybe we could have it the same for all exports…?